src/mpid/ch4/generic/am/mpidig_am_globals.c

```diff
@@ -10,11 +10,10 @@
  */
 
 #include "mpidimpl.h"
-#include "mpidig.h"
 #include "ch4_impl.h"
 #include "mpidch4r.h"
 
-MPIDIG_global_t MPIDIG_global = { {0}
+MPIDIG_am_global_t MPIDIG_am_global = { {0}
```

Comment:
I realized `MPIDIG_global_t` is also AM specific. Eventually there should be an `MPIDIG_global_t`  in `ch4/generic` folder that includes `MPIDIG_am_global`, `MPIDIG_gpu_global`, ...